src/domain/enrolment/enrolmentForm/EnrolmentForm.tsx

```diff
@@ -115,15 +116,28 @@ const EnrolmentForm: React.FC<Props> = ({
   const { t } = useTranslation();
   const locale = useLocale();
 
-  const studyLevelOptions = Object.values(StudyLevel).map((level) => ({
-    label: level.startsWith('GRADE')
-      ? t('enrolment:studyLevel.grade_interval', {
-          postProcess: 'interval',
-          count: Number(level.split('_')[1]),
-        })
-      : translateValue('enrolment:studyLevel.', level, t),
-    value: level,
-  }));
+  const { data: studyLevelsData } = useStudyLevelsQuery();
+
+  const studyLevelOptions = studyLevelsData?.studyLevels?.edges?.map(
+    (studyLevelNodeEdge) => {
+      const level:
+        | string
+        | undefined = studyLevelNodeEdge?.node?.id.toUpperCase();
```

Comment:
defining explicit types here is little bit redundant here maybe?

I think the readability suffers a little bit but doesn't add any benefits :) node.id already has type generated for us.